frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import {
+  // BaremetalHostRole,
+  // BaremetalHostStatus,
+  getName,
+  getNamespace,
+  // getResource,
+} from 'kubevirt-web-ui-components';
+
+// TODO(jtomasek): update import once models are moved to console-shared package
+// import { MachineModel, NodeModel } from '@console/internal/models';
+// import { referenceForModel } from '@console/internal/module/k8s';
+
+import { ListHeader, ColHead, List, ListPage, ResourceRow } from '@console/internal/components/factory';
+
+import { ResourceLink } from '@console/internal/components/utils';
+// import { WithResources } from '@console/shared';
+
+import { BaremetalHostModel } from '../models';
+import { getHostBmcAddress } from '../selectors';
+import MachineCell from './machine-cell';
+
+const nameColumnClasses = 'col-lg-2 col-md-4 col-sm-6 col-xs-6';
+// const statusColumnClasses = 'col-lg-2 col-md-4 hidden-sm hidden-xs';
+const machineColumnClasses = 'col-lg-3 visible-lg';
+// const roleColumnClasses = 'col-lg-2 visible-lg';
+const addressColumnClasses = 'col-lg-2 visible-lg';
```

Comment:
I know this temporary, but columns no longer add up to 12. You might add a column like created to fill things out until we are able to add the others.

Reply:
I've added common column class for the 3 current columns and will adjust it back once remaining columns are added.